src/renderer/settings/appearance-settings-component.ts

```diff
@@ -96,6 +96,46 @@ export const appearanceSettingsComponent = Vue.extend({
                             </div>
                         </div>
                     </div>
+                    <div class="settings__option">
+                        <div class="settings__option-name">{{ translations.appearanceSettingsUserInputRadius }}</div>
+                        <div class="settings__option-content">
+                            <div class="field is-grouped is-grouped-right">
+                                <div class="control">
+                                    <input type="text" step="1" class="input" v-model="config.appearanceOptions.userInputRadius" @change="updateConfig">
```

Comment:
I think you can remove the `step="1"` here. It's a text input, not a number input.

Reply:
Okay